src/Classes/ServiceAPI/MyRadio_User.php

```diff
@@ -785,6 +825,16 @@ public function getRequirePasswordChange()
         return $this->require_password_change;
     }
 
+    public function getSignedGDPR()
+    {
+        return $this->gdpr_accepted;
+    }
+
+    public function getHideProfile()
```

Comment:
Feels like a poor name for a function that returns a bool. `isHidden` ?

Similarly for `getSignedGDPR`